aslam_cv_cameras/src/random-camera-generator.cc

```diff
@@ -0,0 +1,75 @@
+#include "aslam/cameras/random-camera-generator.h"
+
+#include "aslam/cameras/distortion-equidistant.h"
+#include "aslam/cameras/distortion-fisheye.h"
+#include "aslam/cameras/distortion-null.h"
+#include "aslam/cameras/distortion-radtan.h"
+#include "aslam/cameras/distortion.h"
+
+namespace aslam {
+
+NCamera::Ptr createTestNCamera(size_t num_cameras) {
+  std::vector<Camera::Ptr> cameras;
+  Aligned<std::vector, Transformation> T_C_B_vector;
+
+  for (size_t camera_idx = 0u; camera_idx < num_cameras; ++camera_idx) {
+    cameras.push_back(PinholeCamera::createTestCamera<RadTanDistortion>());
+
+    // Offset each camera 0.1 m in x direction and rotate it to face forward.
+    Eigen::Vector3d position(0.1 * (camera_idx + 1), 0.0, 0.0);
+    Quaternion q_C_B(0.5, 0.5, -0.5, 0.5);
+    Transformation T_C_B(q_C_B, position);
+    T_C_B_vector.push_back(T_C_B);
+  }
+
+  NCameraId rig_id;
+  generateId(&rig_id);
+  std::string description("Test camera rig");
```

Comment:
A member of what? And also compared to making it a `consexpr char` somewhere in global scope I feel this is more clear since it's specific to the test function and created close to where it's used.

Reply:
oh you are right. could we not make it a const within the scope of the function? or at the level of the compilation unit?